### ISSUE TITLE
Bug: creating a new carrier from a predefined template and exiting the page without saving still creates the carrier

### ISSUE THREAD

**Issue by javibookline** [jambonz/jambonz-webapp#277]
Steps to reproduce: (tested on jambonz/webapp:0.8.3, jambonz/webapp:0.8.1)

1. go to Carriers page
2. click button "Add Carrier"
3. select a predefined carrier from the list
4. without editing anything, exit the page (either via Cancel button or just leave the page)

A new carrier has been created

This is an issue because new carriers may be accidentally created without notice - this happened to us and gave us some problems.

**Comment by davehorton** [comment]
Yes, this is a bug, but it was done this way because it simplified the implementation.  We may look at this in the future but its not a high priority
